_integration-schemas/zoom/v1/meeting_poll_results.md

```diff
@@ -0,0 +1,44 @@
+---
+tap: "zoom"
+version: "1"
+key: ""
+
+name: "meeting_poll_results"
+doc-link: ""
+singer-schema: "https://github.com/singer-io/tap-zoom/blob/master/tap_zoom/schemas/meeting_poll_results.json"
+description: |
+  The `{{ table.name }}` table contains information about the results of polls from your {{ integration.display_name }} meetings.
+
+replication-method: "Full Table"
+
+api-method:
+    name: "getPastMeetingPolls"
+    doc-link: "https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/listpastmeetingpolls"
+
+attributes:
+  - name: "email"
+    type: "string"
+    primary-key: true
+    description: |
+      The email used for the {{ integration.display_name }} meeting.
+    foreign-key-id: "meeting-email"  
```

Comment:
Is this actually a foreign key? Keep in mind that just because a column is part of a composite Primary Key, it doesn't mean it's necessarily a foreign key.

For example, sometimes dates are part of a composite primary key - let's say a table has a composite PK of `report_id : report_date`. The `report_date` is a PK column, but it's unlikely that it would point to any records outside of that specific table.